migrate2rocky/migrate2rocky.sh

```diff
@@ -173,84 +174,92 @@ os-release () (
 pkg_ver() (
     ver=$(rpm -q --qf '%{VERSION}\n' "$1") || return 2
     if [[ $(sort -V <<<"$ver"$'\n'"$2" | head -1) != "$2" ]]; then
-	return 1
+        return 1
     fi
     return 0
 )
 
 # Set up a temporary directory.
 pre_setup () {
     if ! tmp_dir=$(mktemp -d) || [[ ! -d "$tmp_dir" ]]; then
-	exit_message "Error creating temp dir"
+        exit_message "Error creating temp dir"
     fi
     # failglob makes pathname expansion fail if empty, dotglob adds files
     # starting with . to pathname expansion
     if ( shopt -s failglob dotglob; : "$tmp_dir"/* ) 2>/dev/null ; then
-	exit_message "Temp dir not empty"
+        exit_message "Temp dir not empty"
     fi
 }
 
 # Cleanup function gets rid of the temporary directory.
 exit_clean () {
     if [[ -d "$tmp_dir" ]]; then
-	rm -rf "$tmp_dir"
+        rm -rf "$tmp_dir"
     fi
     if [[ -f "$container_macros" ]]; then
-	rm -f "$container_macros"
+        rm -f "$container_macros"
     fi
 }
 
 pre_check () {
     if [[ -e /etc/rhsm/ca/katello-server-ca.pem ]]; then
-	    exit_message "Migration from Katello-modified systems is not supported by migrate2rocky. See the README file for details."
+        exit_message \
+'Migration from Katello-modified systems is not supported by migrate2rocky. '\
+'See the README file for details.'
     fi
     if [[ -e /etc/salt/minion.d/susemanager.conf ]]; then
-        exit_message "Migration from Uyuni/SUSE Manager-modified systems is not supported by migrate2rocky. See the README file for details."
+        exit_message \
+'Migration from Uyuni/SUSE Manager-modified systems is not supported by '\
+'migrate2rocky. See the README file for details.'
     fi
 }
 
 # All of the binaries used by this script are available in a EL8 minimal install
 # and are in /bin, so we should not encounter a system where the script doesn't
-# work unless it's severly broken.  This is just a simple check that will cause
+# work unless it's severely broken.  This is just a simple check that will cause
 # the script to bail if any expected system utilities are missing.
```

Comment:
... to bail ?
see also:
function:
https://github.com/rocky-linux/rocky-tools/blob/c53d1488fdedb84928eee29d980be96811d8232f/migrate2rocky/migrate2rocky.sh#L358
and 
highlighted in:
https://github.com/rocky-linux/rocky-tools/blob/c53d1488fdedb84928eee29d980be96811d8232f/migrate2rocky/README.md
...is run.  The RockyLinux kernel should come up as the default highlighed kernel...

Reply:
Don't see bail, got function and highlighted.

Reply:
# the script to bail if any expected system utilities are missing.

Reply:
https://github.com/rocky-linux/rocky-tools/blob/7f08a2b69635223e5d8532b30c4580f528558571/migrate2rocky.sh#L62